### ISSUE TITLE
Paste Thumbnail into Publish 

### ISSUE THREAD

**Issue by abitrolly** [observablehq/feedback#219]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I need to save file from online editor, then click upload, then find the file again.

![image](https://user-images.githubusercontent.com/8781107/133573277-1064c9c0-b132-42e6-aedc-1b2c9365517f.png)

**Describe the solution you'd like**
Would be much more convenient if the form allowed to paste the image that is already in the clipboard. It will be more easy to use images from online editors, or from generated art in Observable notebooks.

**Comment by CobusT** [comment]
Interesting idea. In the mean time, you know that you can drag and drop and image into this window?

**Comment by abitrolly** [comment]
No, didn't know that. Probably because I never use this preferring mouseless workflow.

**Comment by mootari** [comment]
You can already paste images from the clipboard though.

@abitrolly If this isn't working for you, mind sharing your OS and browser? It might also help to have a look at the clipboard content that you're trying to paste. To do that ...
1. Open https://observablehq.com/@mootari/look-what-the-cat-dragged-in
2. Click on the grey area that's labeled "Click or drop"
3. Paste via Ctrl+V
4. Share a screenshot of the listed dataTransfer items

**Comment by abitrolly** [comment]
@mootari it works in Firefox 92, indeed. But it works only when the form is just opened. If I change options or the cursor is in custom name field, the image paste doesn't work. Perhaps I'd try to click the placeholder element if the Thumbnail hint suggested that paste is possible.

On the debug page https://observablehq.com/@mootari/look-what-the-cat-dragged-in there is no option to copy canvas image. Only open it in a new tab.
